### PR TITLE
Preparations for modbus support on S series pumps

### PR DESCRIPTION
**Pull request by elupus**

### CODE REVIEW

**Comment by elupus** [comment]
The csv's don't seem to include any mapping information sadly.

File: nibe/connection/modbus.py [hunk 1]
```diff
@@ -20,13 +33,39 @@ def __init__(self, heatpump: HeatPump, url, slave_id, conn_options=None):
...
+
+            with async_timeout.timeout(timeout):
+                if entity_type == 4:
+                    result = await self._client.read_input_registers(
+                        slave_id=self._slave_id,
+                        starting_address=entity_number,
+                        quantity=entity_count,
+                    )
+                elif entity_type == 3:
+                    result = await self._client.read_holding_registers(
```
Comment (yozik04) [inline nibe/connection/modbus.py:47]: I’d create a mapping from entity type to a method that reads register/input/coil.

Comment (elupus) [reply nibe/connection/modbus.py:47]: It get a bit ugly since it's member functions.

**Comment by yozik04** [comment]
Is this ready? Should I merge?

**Comment by elupus** [comment]
Would like to get an english registry set. But it should not break anything to merge.

**Comment by elupus** [comment]
I think we can merge this as is, we can replace the DB with an english one once we have it. This make it easier to test on a real unit.

**Pull request merged**